admob/app/src/main/AndroidManifest.xml

```diff
@@ -8,15 +8,14 @@
         android:label="@string/app_name"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
-        <activity android:name=".MainActivity">
+        <activity android:name="com.google.firebase.example.admob.MainActivity">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
 
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
-        <activity android:name=".KotlinMainActivity" />
```

Comment:
Oops, looks like I had a lot of copypasta errors from adding all the new modules.  Thanks!